### DISCUSSION TITLE
Why can't rider attach its debugger?

### DISCUSSION THREAD

**Discussion by StefanH-AT** [MikeSchulze/gdUnit4#425] [Q&A] [answered]
I know I know, this is documented. https://mikeschulze.github.io/gdUnit4/faq/vstest-adapter/#test-debug-workaround-for-jetbrains-rider

Just wondering why this is needed? Why can't rider attach its debugger? Is there an issue open for this?

Also the workaround doesn't seem to work for me :/ Loop doesn't seem to get cancelled according to my whirring cpu fan

**Comment by MikeSchulze** [comment, marked as answer]
This is a rider issue, it does not support the VSTest `IFrameworkHandle2` where is need to attach the debugger.

https://youtrack.jetbrains.com/issue/RSRP-496314

**Reply by StefanH-AT** [reply]
Thank you!

**Comment by MikeSchulze** [comment]
It is solved with GdUnit4Net version v4.3.1
